lib/spack/spack/test/cmd/install.py

```diff
@@ -741,6 +746,7 @@ def test_install_deps_then_package(tmpdir, mock_fetch, install_mockery):
     assert os.path.exists(root.prefix)
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="does not run on windows")
```

Comment:
I know a lot of our tests are skipped for the "does not run on Windows" reason, but if we're adding new skips, can we add why? If it's a lot of work to determine why, then this is fine (which is why there are so many of the vague reason string), but ideally we'd be able to offer more of an explanation so we can know when we can turn the tests back on.